pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -25,6 +28,9 @@ import (
 var (
 	serializer     = scheme.Codecs.LegacyCodec(configv1.SchemeGroupVersion)
 	kubeSerializer = kubescheme.Codecs.LegacyCodec(corev1.SchemeGroupVersion)
+
+	// maxEventTimeInterval represents the "only keep events that are maximum 1h old"
+	maxEventTimeInterval = 1 * time.Hour
```

Comment:
Technically this needs to be dynamic based on the interval - that would be the right thing to do.  Can you just add a TODO here for `TODO: make this dynamic like the reporting window based on configured interval`?  The interval is available via a getter for the reporter function anyway.

Reply:
todo added